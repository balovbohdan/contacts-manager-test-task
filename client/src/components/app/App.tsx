import * as React from 'react';

import {Redux} from './Redux';
import {Apollo} from './Apollo';
import {Content} from './content';

export const App = () =>
    <Apollo>
        <Redux>
            <Content/>
        </Redux>
    </Apollo>;