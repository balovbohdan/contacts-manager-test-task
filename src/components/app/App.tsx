import * as React from 'react';

import {Header} from '@components/header';

import {Router} from './Router';
import {StoreProvider} from './StoreProvider';

export const App = () =>
    <StoreProvider>
        <Router>
            <Header/>
        </Router>
    </StoreProvider>;