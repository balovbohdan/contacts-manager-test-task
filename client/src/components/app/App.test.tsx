import {shallow} from 'enzyme';
import * as React from 'react';

import {App} from './App';

describe('Root App', () => {
    it('Should match snapshot', () => {
        const app = shallow(<App/>);

        expect(app).toMatchSnapshot();
    });
});