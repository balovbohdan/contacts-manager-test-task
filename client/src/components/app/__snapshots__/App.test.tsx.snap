// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root App Should match snapshot 1`] = `
<Component>
  <Component>
    <Connect(C) />
  </Component>
</Component>
`;
